src/App.js

```diff
@@ -1,26 +1,32 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovies] = useState(data);
 
-  render() {
-    const { movies } = this.state;
+  function addMovie(newMovie, showStatus) {
+    const repeatCheck = movies.some(
+      movie => movie.imdbID === newMovie.imdbID,
+    );
+
+    if (newMovie && showStatus && !repeatCheck) {
+      setMovies(prevList => [newMovie, ...prevList]);
+    }
+  }
```

Comment:
```suggestion
    const isMovieUnique = movies.every(
      movie => movie.imdbID !== newMovie.imdbID,
    );

    if (newMovie && showStatus && isMovieUnique) {
      setMovies(prevList => [newMovie, ...prevList]);
    }
  }
```
It's better to check if the found movie is unique, because it's easier to read the condition 'if something is true' than 'if something is false'